src/foam/u2/dialog/NotificationMessage.js

```diff
@@ -20,111 +20,204 @@ foam.CLASS({
-    ^ .close-x:after {
-      transform: rotate(-45deg);
+    ^error-content {
+      color: #631414;
+    }
+    ^warning-content {
+      color: #816819;
+    }
+    ^info-content {
+      color: #202341;
+    }
+    ^link-icon {
+      display: inline-block;
+      margin-top: 2px;
+      vertical-align: middle;
+      margin-right: 0 !important;
+      width: 16px;
+      height: 16px;
+    }
+    ^close-icon {
+      background-image: url("images/round-close-icon.svg");
+      background-size: 12px 12px;
+      cursor:pointer;
+      height: 12px;
+      opacity: 0.5;
+      width: 12px;
+      position: absolute;
+      top: 18;
+    }
+    ^close-icon:hover {
+      opacity: 1;
     }
   `,
 
+  properties: [
+    {
+      class: 'String',
+      name: 'type'
+    },
+    'message',
+    'description'
+  ],
+
   methods: [
+
     function initE() {
       var self = this;
 
+      var img;
+      if ( this.type === 'error' ) {
+        img = 'images/round-error-red.svg';
+      } else if ( this.type === 'warning' ) {
+        img = 'images/baseline-warning-yellow.svg';
+      } else if ( this.type === 'info' ) {
+        img = 'images/round-error-blue.svg';
+      } else {
+        img = 'images/round-check-circle-green.svg';
+      }
       this
         .addClass(this.myClass())
-        .enableClass(this.myClass('error-background'), this.type === 'error')
-        .start()
-          .callIfElse(foam.String.isInstance(this.message), function() {
-            this.add(self.message);
-          }, function() {
-            this.tag(self.message);
-          })
-        .end()
-        .startContext({ data: this })
-          .start().addClass('close-x').add(this.CLOSE).end()
-        .endContext();
+        .start().addClass(this.myClass('inner'))
+          .enableClass(this.myClass('error-background'), this.type === 'error')
+          .enableClass(this.myClass('warning-background'), this.type === 'warning')
+          .enableClass(this.myClass('info-background'), this.type === 'info')
+          .start('div').addClass(this.myClass('banner'))
+            .enableClass(this.myClass('error-banner'), this.type === 'error')
+            .enableClass(this.myClass('warning-banner'), this.type === 'warning')
+            .enableClass(this.myClass('info-banner'), this.type === 'info')
+          .end()
+          .start()
+            .start('img')
+              .addClass(this.myClass('status-icon'))
+              .attrs({ src: img })
+            .end()
+            .start().addClass(this.myClass('content'))
+              .enableClass(this.myClass('error-content'), this.type === 'error')
+              .enableClass(this.myClass('warning-content'), this.type === 'warning')
+              .enableClass(this.myClass('info-content'), this.type === 'info')
+              .callIfElse(foam.String.isInstance(this.message), function() {
+                this.add(self.message);
+              }, function() {
+                this.tag(self.message);
```

Comment:
add a console.log(self.message); below this tag .. and maybe below the description below as well if it is not empty.